api/match.py

```
@@ -0,0 +1,127 @@
+        'stadium': stadium,
+        'home_team_id': home_team_id,
+        'home_team_name': home_team_name,
+        'home_team_score': game.get('homeTeamScore'),
+        'away_team_id': away_team_id,
+        'away_team_name': away_team_name,
+        'away_team_score': game.get('awayTeamScore'),
+        'status_code': game.get('statusCode'),
+        'winner': game.get('winner'),
+    }
+
+def get_match_preview_info(game_id):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)